File: app/src/main/java/com/example/testing/ui/notHome/NotHomeViewModel.java
package com.example.testing.ui.notHome;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class NotHomeViewModel extends ViewModel {

    public NotHomeViewModel() {
        MutableLiveData<String> mText = new MutableLiveData<>();
        mText.setValue("This is a brand new Fragment");
    }

}

File: app/src/main/java/com/example/testing/ui/notHome/NotHomeViewModel.kt
package com.example.testing.ui.notHome

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class NotHomeViewModel : ViewModel() {

    private val _text = MutableLiveData<String>().apply {
        value = "This is a brand new Fragment"
    }
    val text: LiveData<String> = _text
}